spec/openapi.yaml

```diff
@@ -8683,13 +8683,23 @@ paths:
               - Review
             
               - Post-Edit
+            
+              - Monolingual review
```

Comment:
We don't need Monolingual review, it's not used since 2019

Reply:
removed